flake8_routable.py

```diff
@@ -153,6 +154,9 @@ def visit_ImportFrom(self, node: ast.ImportFrom) -> None:
         if node.level > 0:
             self.errors.append((node.lineno, node.col_offset, ROU106))
 
+        if node.module is not None and "models." in node.module:
```

Comment:
I don't imagine having any module called `not_a_model`, but do be safe and strict with the match, can we prepend a `.`?

```suggestion
        if node.module is not None and ".models." in node.module:
```

Reply:
Love it, fixed here: https://github.com/routablehq/flake8-routable/pull/15/commits/6f2034668da17cc41c38311531daa761a1ad0960